package.json

```diff
@@ -1,22 +1,22 @@
 {
   "name": "xmldom",
-  "version": "0.1.27",
+  "version": "0.1.28",

```

Comment:
I think not because @jeremlb told me they refused a PR from @lseguin42 so we forked the project.